src/actor/character/styles/_character.scss

```
@@ -47,3 +47,81 @@
     cursor: pointer;
   }
 }
+
+.tooltip-top, .tooltip-bottom, .tooltip-right, .tooltip-left {
+  position: relative;
+  display: inline-block;
+  border-bottom: 1px dotted black;
+}
+
+.tooltip-top .tooltip-text {
+  bottom: 100%;
+  left: 50%;
+}
+
+.tooltip-bottom .tooltip-text {
+  top: 100%;
+  left: 50%;
+}
+
+.tooltip-right .tooltip-text {
+  top: -5px;
+  left: 105%;
+}
+
+.tooltip-left .tooltip-text {
+  top: -5px;
+  right: 105%;
+}
+
+:is(.tooltip-top, .tooltip-bottom, .tooltip-right, .tooltip-left) .tooltip-text {
+  visibility: hidden;
+  background-color: black;
+  color: #fff;
+  padding: 5px 0;
```

The left and right of the text look a wee bit tight. Maybe
```suggestion
  padding: 5px;
```